metrics/prometheus/kubernetes.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: prometheus
+    spec:
+      containers:
+      - name: prometheus
+        image: prom/prometheus:v2.0.0
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: prometheus
+          containerPort: 9090
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /prometheus
+          name: prometheus-data
+        - mountPath: /etc/prometheus
+          name: prometheus-config
+      volumes:
+        - name: prometheus-data
+          emptyDir: {}
+        - name: prometheus-config
+          configMap:
+            name: prometheus-config
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-config
+data:
+  prometheus.yml: |
+    global:
+      scrape_interval: 15s
+      evaluation_interval: 15s
+    scrape_configs:
+      - job_name: 'kafka_job'
```

Comment:
The job_name is `kafka_job` ... but you don't specify any Kafka pods or anything like that. It will scrape all pods in the cluster, or?

Reply:
It refers to be part of getting metrics for a Kafka cluster, or ?

Reply:
Yeah, but it technicaly gets metrics from every pod which would run in the same sluster, or?

Reply:
Changed to `pods_job` ;)